src/components/BatchSearchFilterDropdown.vue

```diff
@@ -32,6 +34,10 @@ export default {
       type: String,
       required: true
     },
+    eq: {
+      type: Function,
```

Comment:
Why did you pass a function instead of a boolean here?

Reply:
It's related to the behavior of selectable-dropdown in murmur:
https://github.com/ICIJ/murmur/blob/a9a77a00e36811d8a0e10cfb2754506cda6a2774/lib/components/SelectableDropdown.vue#L112-L118

The equality function is relative to one item at a time that's why it passes down the function as a prop. I don't see how it can be computed in advance on a upper level.

Reply:
Ah, yes, Murmur! I understand now. Makes more sens, thanks!